include/drivers/hwinfo.h

```diff
@@ -30,6 +30,18 @@
 extern "C" {
 #endif
 
+enum reset_cause {
```

Comment:
This should be a set of `#define` statements instead of an `enum`

Reply:
This old comment hasn't been addressed yet. We cannot use enum here, `cause` parameter is of type `uint32_t*`.